blockchain/blockchain-wallet-v4-frontend
# Feature/verify message

## Description
Add "Verify Message" modal to validate the signature of a Bitcoin message.

## Change Type
Please enter one or more of the following: 
- Feature

## Testing Steps
Detail the steps required for the reviewer(s) to verify and test these changes.

1.  Click `Settings`
1.  Click `Wallets & Addresses`.
1.  Click `Verify Message`.
1.  Enter a valid address, message, and signature, e.g., `15WJg3bnKkuzeLxCKUGCKjRM42d3LgWg9u`, `Fun is nuf spelled backwards!`, and `HxB1FMYnYLda3SDhFVXpfyUfxfGOUOuCkrY0rwxIFaXzWgv/y7Wlijd9C/2t6ydoVOpi4v5XedgvDDOb8a6ZCfo=`.
1.  You should see a message indicating a valid signature.
1.  Try corrupting any of the three fields and watch the message change.

## Code Checklist
